imports/plugins/core/system-info/server/no-meteor/schemas/schema.graphql

```diff
@@ -0,0 +1,16 @@
+extend type Query {
+  systemInformation: SystemInfo
+}
+
+"Represents Reaction Plugin"
+type Plugin {
+  name: String,
```

Comment:
Every field should have a string above it with the field description. These are shown in GraphQL Playground when viewing the schema docs.

Also, I think the commas are ignored, but they aren't necessary and we don't include them.